mmv1/products/compute/api.yaml

```diff
@@ -12108,6 +12108,66 @@ objects:
+          - !ruby/object:Api::Type::NestedObject
+            name: 'urlRewrite'
+            description : |
+              The spec to modify the URL of the request, before forwarding the request to the matched service.
+              urlRewrite is the only action supported in UrlMaps for external HTTP(S) load balancers.
+              Not supported when the URL map is bound to a target gRPC proxy that has the validateForProxyless field set to true.
+            at_least_one_of:
+              - default_route_action.0.weighted_backend_services
+              - default_route_action.0.url_rewrite
+              - default_route_action.0.timeout
+              - default_route_action.0.retry_policy
+              - default_route_action.0.request_mirror_policy
+              - default_route_action.0.cors_policy
+              - default_route_action.0.fault_injection_policy
+            properties:
+              - !ruby/object:Api::Type::String
+                name: 'pathPrefixRewrite'
+                description: |
+                  Before forwarding the request to the selected backend service, the matching portion of the request's path is replaced by pathPrefixRewrite.
+                  The value must be from 1 to 1024 characters.
+                at_least_one_of:
+                  - default_route_action.0.url_rewrite.0.path_prefix_rewrite
+                  - default_route_action.0.url_rewrite.0.host_rewrite
+              - !ruby/object:Api::Type::String
+                name: 'hostRewrite'
+                description: |
+                  Before forwarding the request to the selected service, the request's host header is replaced with contents of hostRewrite.
+                  The value must be from 1 to 255 characters.
+                at_least_one_of:
+                  - default_route_action.0.url_rewrite.0.path_prefix_rewrite
+                  - default_route_action.0.url_rewrite.0.host_rewrite
+          - !ruby/object:Api::Type::NestedObject
+            name: 'timeout'
+            description: |
+              Specifies the timeout for the selected route. Timeout is computed from the time the request has been fully processed (known as end-of-stream) up until the response has been processed. Timeout includes all retries.
+              If not specified, this field uses the largest timeout among all backend services associated with the route.
+              Not supported when the URL map is bound to a target gRPC proxy that has validateForProxyless field set to true.
+            at_least_one_of:
+              - default_route_action.0.weighted_backend_services
+              - default_route_action.0.url_rewrite
+              - default_route_action.0.timeout
+              - default_route_action.0.retry_policy
+              - default_route_action.0.request_mirror_policy
+              - default_route_action.0.cors_policy
+              - default_route_action.0.fault_injection_policy
```

Comment:
Note that the top-level properties that weren't added in this pr also need these values. I notice they are commented out.

Reply:
Thanks for catching this - I've uncommented them and added a missing `at_least_one_of` block too